spec/support/sass_support.rb

```diff
@@ -0,0 +1,10 @@
+module SassSupport
+  def generate_css
+    _mkdir('tmp')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.